### DISCUSSION TITLE
Add "Copy Command" to install all components used in a block

### DISCUSSION THREAD

**Discussion by jolbol1** [shadcn-ui/ui#3288] [Ideas]
It would be a great enhancement to include a "Copy Command" button for each block. This button would allow users to quickly copy the necessary command to install all the required components for a specific block.

For instance, if a user wants to set up a project using the components from "Dashboard 7", they could simply click the "Copy Command" button, which would copy the following command to their clipboard:

```sh
pnpm dlx shadcn-ui@latest add badge button breadcrumb card dropdown-menu input pagination progress separator sheet table tabs tooltip
```
The user can then easily paste this command into their terminal to install all the required components in one go.

**Comment by onurhan1337** [comment]
Good point. I'll try to develop this. I'll update here when I create a PR about that.

> You can follow up here https://github.com/shadcn-ui/ui/pull/3365
